CONTRIBUTING.md

```diff
@@ -46,14 +37,16 @@ For now, the format of each page has to match the following:
 
 - example description
 
-`command -arg1 -arg2`
+`command -opt1 -opt2 -arg1 {{arg_value}}`
 
 - example description
 
-`command -arg1 -arg2`
+`command -opt1 -opt2`
 ```
 
-Eventually we might relax the format to accept any Markdown, but for now this has the advantage of adding some consitency between all pages, and making sure we focus on concrete examples rather than lengthy explanation of the different flags.
+User-provided values should use the `{{token}}` syntax, to allow clients to highlight them. For example: `tar cf {{file}}`
+
+Eventually we might relax the format to accept any Markdown, but for now this has the advantage of adding some consistency between all pages, and making sure we focus on concrete examples rather than lengthy explanation of the different flags.
```

Comment:
I'd actually drop this line, it's very unlikely that we'll relax the format given the number of clients processing the current pages.